Developers/Unity Project/Spooksville/Assets/Scripts/DialogManager.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+
+public class DialogManager : MonoBehaviour
+{
+    public static DialogManager instance;
+
+    public Text container;
+    private List<string> collectionMessages = new List<string>();
+
+    private void Start()
+    {
+        if (instance == null)
+            instance = this;
+
+        collectionMessages.Add("Get out of here!");
+    }
+
+    private void Awake()
+    {
+        container.gameObject.SetActive(false);
+    }
+
+    public void DisplayRandomCollectText()
+    {
+        string text = collectionMessages[Mathf.RoundToInt(Random.Range(0, collectionMessages.Count - 1))];
+
+        RenderText(text, 3f);
+    }
+    public void RenderText(string text)
+    {
+        GetTextContainer().gameObject.SetActive(true);
+        GetTextContainer().text = text;
+    }
+
+    public void RenderText(string text, float time)
+    {
+        GetTextContainer().gameObject.SetActive(true);
+        GetTextContainer().text = text;
+
+        Invoke("DeactivateTextContainer", time);
```

Comment:
This is fine for such a simple use case but you may want to look into coroutines if you haven't used those yet. They are type-safe, allow parameters to be passed, and permit more complex logic sequences to occur over time.